bookshelf/composer.json

```diff
@@ -1,6 +1,6 @@
 {
   "require": {
-    "php": ">=7",
+    "php": "^7.1.3",
```

Comment:
^7.1.3 is the equivalent to: >= 7.1.3 < 8.0.0
That makes sense to me.

Reply:
What makes less sense: why does specifying 7.1.3 in the composer.json file help? Does the buildpack deliberately setup the minimum compatible PHP version?

Reply:
Async: @rogerthatdev says this is indeed end-to-end tested as working with the PHP buildpack. I'll have to learn more about how this works later, fixing the bug++